ios-base/DroidKaigi 2020/SideBar/Views/SidebarViewController.swift

```diff
@@ -0,0 +1,61 @@
+import Material
+import UIKit
+
+enum SwitchViewControllerType: Int {
+    case timeline
+    case about
+    case info
+    case map
+    case sponsor
+    case contributor
+    case setting
+}
+
+final class SidebarViewController: UITableViewController {
+    weak var rootViewController: UIViewController?
+
+    static func instantiate(rootViewController: UIViewController) -> SidebarViewController {
+        guard let viewController = UIStoryboard(name: "SidebarViewController", bundle: .main).instantiateInitialViewController() as? SidebarViewController else { fatalError() }
+        viewController.rootViewController = rootViewController
+        return viewController
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.tableHeaderView?.frame.size.height = 190
+        tableView.rowHeight = 65
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        // select default cell (timeline)
+        let indexPath = IndexPath(row: 0, section: 0)
```

Comment:
I think it would be more explicit to use `timeline.rawValue`.

Reply:
Exactly, I think so too. I will fix it 👍